lib/tasks/create_modify_providers.rake

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rake'
+require 'json'
+
+def display_error_msg(current_provider)
+  error_msg = ''
+  if current_provider.errors.nil?
+    error_msg = 'Unknown Error'
+  else
+    current_provider.errors.messages.each do |k, v|
+      v.each do |msg|
+        error_msg += "#{k} #{msg}\n"
+      end
+    end
+  end
+  puts "Error! Could not add/update provider #{each_provider[name]}. Exited with message(s): #{error_msg}"
+end
+
+namespace :provider do
+  desc 'Adding providers if they do not exist or modifying them overall'
+  task :load, %i[file] => :environment do |_t, args|
+    all_providers = JSON.parse(File.open(args.file, 'r:UTF-8').read)['Entries']
+    all_providers.each do |each_provider|
+      specific_provider = Provider.where(name: each_provider['name']).take
+      each_provider['provider_type'] = 'smart_epic'
```

Comment:
provider type should be part of the JSON, not part of the load functionality.  This precludes the use of other provider types

Reply:
Looking at the data file the provider type is already included in that too.